src/PowerSimulations.jl

```diff
@@ -3,120 +3,104 @@ module PowerSimulations
 #################################################################################
 # Exports
 
-#Core Exports
-export PowerSimulationsModel
-export PowerResults
+#Network Relevant Exports
+export StandardPTDFForm
+export CopperPlatePowerModel
 
-#Base Modeling Exports
-export CustomModel
-export EconomicDispatch
+#operation_models
 export UnitCommitment
+export EconomicDispatch
+export SCEconomicDispatch
+export OptimalPowerFlow
 
-#Device formulation Export
-
-
-#Network Relevant Exports
-export AbstractDCPowerModel
-export CopperPlatePowerModel
+#functions
+export solve_op_model!
 
-#Functions
-export buildmodel!
-export simulatemodel
 
 #################################################################################
 # Imports
-using JuMP
-using TimeSeries
-using PowerSystems
+import JuMP
+#using TimeSeries
+import PowerSystems
 import PowerModels
-import InfrastructureModels
-using GLPK
-using MathOptInterface
-#using Clp
-#using Cbc
-using Ipopt
-using DataFrames
-using LinearAlgebra
-using LinearAlgebra.BLAS
-using AxisArrays
-using Dates
+import MathOptInterface
+import DataFrames
+import LinearAlgebra
+#import LinearAlgebra.BLAS #needed for the simulation stage
+import AxisArrays
+import Dates
 
 #################################################################################
 # Type Alias From other Packages
 const PM = PowerModels
-const IM = InfrastructureModels
-const NetworkModel = PM.AbstractPowerFormulation
-const PS = PowerSimulations
+const PSY = PowerSystems
+const PSI = PowerSimulations
 const MOI = MathOptInterface
 const MOIU = MathOptInterface.Utilities
-const PTDFArray = AxisArrays.AxisArray{Float64,2,Array{Float64,2},Tuple{AxisArrays.Axis{:branches,Array{String,1}},AxisArrays.Axis{:buses,Array{String,1}}}}
 
 #Type Alias for JuMP containers
-const JumpVariable = JuMP.JuMPArray{JuMP.VariableRef,2,Tuple{Array{String,1},UnitRange{Int64}}}
 const JumpExpressionMatrix = Matrix{<:JuMP.GenericAffExpr}
-const JumpAffineExpressionArray = Array{JuMP.GenericAffExpr{Float64,JuMP.VariableRef},2}
-const BalanceNamedTuple = NamedTuple{(:var_active, :var_reactive, :timeseries_active, :timeseries_reactive),Tuple{JumpAffineExpressionArray, UJ, Array{Float64,2}, UF}} where {UJ <: Union{Nothing,JumpAffineExpressionArray}, UF <: Union{Nothing, Array{Float64,2}}}
+const JumpAffineExpressionArray = Array{JuMP.GenericAffExpr{Float64,V},2} where V <: JuMP.AbstractVariableRef
```

Comment:
Make `Jump` to `JuMP` to be consistent with the rest of the package:

```
const JuMPExpressionMatrix = Matrix{<:JuMP.GenericAffExpr}
const JuMPAffineExpressionArray = Array{JuMP.GenericAffExpr{Float64,V},2} where V <: JuMP.AbstractVariableRef
```

Reply:
fixed via a351de5